### ISSUE TITLE
Allow returning to previous uri

### ISSUE THREAD

**Issue by dibenso** [htty/htty#133]
It would be useful to have a command that is capable of taking the current request uri to the previous one. Like how the back button works in a browser.

**Comment by njonsson** [comment]
Fair enough. I think a `back` command ought to be synonymous with `reuse N` where _N_ is the current `history` index minus 1. In this way, `back` would not issue a request. You could go `back` multiple times, essentially traversing the address and headers of your `history`. It might make the most sense to disable the `response-*` commands after a `back` until another request is made, in order to avoid confusion.

Your thoughts?

**Comment by dibenso** [comment]
Sounds good with me. There would be no reason to get a new response when it already exists in history

**Comment by dibenso** [comment]
Sounds good with me. You're right, there would be no reason to get a new response when it already exists in history.

**Comment by presskey** [comment]
It would be good to create separate ticket for ``*-response`` commands behavior. At the moment after ``reuse`` they still return data of last request with response when it makes more sense to use response of   reused request or return nil at all.
